tests/test_IbetExchange.py

```diff
@@ -78,54 +75,54 @@ class TestTokenFallback:
     #######################################
 
     # Normal_1
-    def test_normal_1(self, users, membership_exchange):
+    def test_normal_1(self, users, exchange):
         _issuer = users['issuer']
         _value = 100
 
         # issue token
-        deploy_args = init_args(membership_exchange.address)
+        deploy_args = init_args(exchange.address)
         membership_token = deploy(users, deploy_args)
```

Comment:
this is standard token(Same for other tests)

Reply:
That's right. I have corrected it.